src/App.tsx

```diff
@@ -1,61 +1,144 @@
+import { FC, ChangeEvent, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserId(userId: number): User | null {
+  return (usersFromServer.find(user => (user.id === userId)) || null);
+}
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserId(todo.userId),
+}));
+
+export const App: FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
```

Comment:
you set just `boolean` type, not a string
fix in all cases
```suggestion
  const [IsTitleError, setIsTitleError] = useState(false);
```